tests/conftest.py

```diff
@@ -0,0 +1,631 @@
+
+
+@pytest.fixture
+def create_test_configuration_setting():
+    """
+    Returns a constructor function for creating a ConfigurationSetting object.
+
+    The calling function should clean up created ConfigurationSetting objects.
+
+    Example:
+
+        def test_something(db_session, create_test_configuration_setting):
+            my_setting = create_test_configuration_setting(db_session)
+
+            [...test body...]
+
+            db_session.delete(my_setting)
+            db_session.commit()
+
+    """
+    def _create_test_configuration_setting(
+        db_session, key=None, value=None, library=None, external_integration=None, **kwargs
+    ):
+        create_kwargs = {"key": key or str(uuid.uuid4())}
+
+        if isinstance(library, Library):
+            create_kwargs['library'] = library
+
+        if isinstance(external_integration, ExternalIntegration):
+            create_kwargs['external_integration'] = external_integration
+
+        if value:
+            create_kwargs['value'] = str(value)
+
+        (setting, _) = get_one_or_create(db_session, ConfigurationSetting, **create_kwargs)
+
+        db_session.commit()
+        return setting
+
+    return _create_test_configuration_setting
+
+
+##############################################################################
+# Places
+##############################################################################
+
+@pytest.fixture
+def crude_us(db_session, create_test_place):
+    """
+    A Place representing the United States. Unlike other Places in this series, this is
+    backed by a crude GeoJSON drawing of the continental United States, not the much more
+    complex GeoJSON that would be obtained from an official source. This shape includes
+    large chunks of ocean, as well as portions of Canada and Mexico.
+    """
+    place = create_test_place(
+        db_session, external_id="US", external_name="United States", place_type=Place.NATION,
+        abbreviated_name="US", parent=None, geometry=(TEST_DATA_DIR / 'crude_us_geojson.json').read_text()
+    )
+    db_session.commit()
+    yield place
+    db_session.delete(place)
+    db_session.commit()
+
+
+@pytest.fixture
+def new_york_state(db_session, create_test_place, crude_us):
+    place = create_test_place(
+        db_session, external_id="36", external_name="New York", place_type=Place.STATE,
+        abbreviated_name="NY", parent=crude_us,
+        geometry=(TEST_DATA_DIR / 'ny_state_geojson.json').read_text()
+    )
+    db_session.commit()
+    yield place
+    db_session.delete(place)
+    db_session.commit()
+
+
+@pytest.fixture
+def connecticut_state(db_session, create_test_place, crude_us):
+    place = create_test_place(
+        db_session, external_id="09", external_name="Connecticut", place_type=Place.STATE,
+        abbreviated_name="CT", parent=crude_us,
+        geometry=(TEST_DATA_DIR / 'ct_state_geojson.json').read_text()
+    )
+    db_session.commit()
+    yield place
```

Comment:
Some of these take a long time to stick into the database (because the GeoJSON is complex), so you might think about scoping them to the test session.